exercises/scale-generator/example.go

```diff
@@ -0,0 +1,68 @@
+// Package scale provides a sample implementation
+package scale
+
+import "strings"
+
+var chromaticScale = []string{"C", "C#", "D", "D#", "E", "F", "F#", "G", "G#", "A",
+	"A#", "B"}
+var flatChromaticScale = []string{"C", "Db", "D", "Eb", "E", "F", "Gb", "G", "Ab",
+	"A", "Bb", "B"}
+var flatKeys = []string{"F", "Bb", "Eb", "Ab", "Db", "Gb", "d", "g", "c", "f", "bb",
+	"eb"}
+
+// Scale returns a type of scale based on the inputs
+func Scale(tonic, interval string) []string {
+	if interval == "" {
+		interval = strings.Repeat("m", 12)
+	}
+	ft := formatTonic(tonic)
+	// chromatic scale
+	if flatKey(tonic, flatKeys) {
+		start := findStart(ft, flatChromaticScale)
+		return printScale(ft, interval, start, flatChromaticScale)
+	}
```

Comment:
instead setting correct slice and calling `printScale` once might be less repetition.